frontend/src/app/testQueue/AoEForm/AoEForm.tsx

```diff
@@ -139,11 +139,6 @@ const AoEForm: React.FC<Props> = ({
 
   const patientIsOver18 = moment().diff(patient.birthDate, "years") >= 18;
 
-  // Null is OK and preferred over an empty string
-  if (lastTest?.dateTested) {
```

Comment:
**question:** So is the idea now that `lastTest.dateTested` is always present and correctly formatted?

Reply:
It's always coming from the backend, so it should be consistent. Furthermore, if you look at how `lastTest.dateTested` is used (it's only used in `PriorTestInputs.tsx`), it's either only used as a truthy/falsey value or it's passed to a formatter